### PR TITLE
Update changelog.

### PR DESCRIPTION
**Pull request by sean-mcmanus**

### CODE REVIEW

**Comment by sean-mcmanus** [comment]
I was planning to post a 1.9.6-prerelease1 on GitHub to verify if it's fixed or not before releasing 1.9.6...assuming I can't find a repro.

File: Extension/CHANGELOG.md [hunk 1]
```diff
@@ -1,16 +1,20 @@
 # C/C++ for Visual Studio Code Change Log
 
+## Version 1.9.6: March 18, 2022
+### Bug Fix
+* Redo 1.9.4 changes with a fix for issue [#9041](https://github.com/microsoft/vscode-cpptools/issues/9041)
+
 ## Version 1.9.5: March 17, 2022
-## Bug Fixes
+### Bug Fixes
 * Undo the 1.9.4 changes to fix the file corruption issue. [#9041](https://github.com/microsoft/vscode-cpptools/issues/9041).
```
Comment (jogo-) [inline Extension/CHANGELOG.md]: Incorrect trailing '.' after links

Comment (jogo-) [inline Extension/CHANGELOG.md]: Incorrect trailing '.' after links

Comment (sean-mcmanus) [reply Extension/CHANGELOG.md]: Thanks.

Comment (sean-mcmanus) [reply Extension/CHANGELOG.md]: I added a trailing period of the 1.9.6 change because that's an usual case where it's referenced directly in a sentence.

**Review by michelleangela — APPROVED** [review]

**Pull request merged**